src/main/java/chess/model/board/ChessBoard.java

```diff
@@ -0,0 +1,65 @@
+package chess.model.board;
+
+import chess.model.piece.Blank;
+import chess.model.piece.Piece;
+import chess.model.position.ChessPosition;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import static java.util.Collections.unmodifiableMap;
+
+public class ChessBoard {
+    private final Map<ChessPosition, Piece> board;
+
+    public ChessBoard(Map<ChessPosition, Piece> board) {
+        this.board = new HashMap<>(board);
+    }
+
+    public void move(ChessPosition sourcePosition, ChessPosition targetPosition) {
+        Piece sourcePiece = board.get(sourcePosition);
+        validateSourceIsNull(sourcePiece);
+        Piece targetPiece = board.get(targetPosition);
+        List<ChessPosition> path = sourcePiece.findPath(sourcePosition, targetPosition, targetPiece);
+        validatePathIsEmpty(path);
+        validatePathContainsPiece(path);
+        replacePiece(sourcePiece, sourcePosition, targetPosition);
+    }
+
+    private void validateSourceIsNull(Piece sourcePiece) {
```

Comment:
```suggestion
    private void validateSourceIsBlank(Piece sourcePiece) {
```
Null보단 잘만들어주신 Blank로 표현하는게 좋아보이네요 :)